crates/sui-framework/packages/sui-framework/sources/accumulator.move

```diff
@@ -17,3 +21,115 @@ fun create(ctx: &TxContext) {
         id: object::sui_accumulator_root_object_id(),
     })
 }
+
+/// Balance object methods
+
+/// The key type to look up a balance object.
+public struct AccumulatorName<phantom T> has copy, drop, store {
+    address: address,
+}
+
+fun accumulator_root_has_balance<K, V: store>(
+    accumulator_root: &AccumulatorRoot,
+    name: AccumulatorName<K>,
+): bool {
+    dynamic_field::exists_with_type<AccumulatorName<K>, V>(&accumulator_root.id, name)
+}
+
+use fun accumulator_root_has_balance as AccumulatorRoot.has_balance;
+
+fun accumulator_root_add_balance<K, V: store>(
+    accumulator_root: &mut AccumulatorRoot,
+    name: AccumulatorName<K>,
+    value: V,
+) {
+    dynamic_field::add(&mut accumulator_root.id, name, value);
```

Comment:
What is `V` here? Many questions depending on that answer 😜 but I'll leave it there for now

Reply:
This function is creating a new accumulator field, so `V` will be `U128` for now. Later, it could be `2U128` or any other accumulator storage type we might need later